algoliasearch/src/main/java/com/algolia/search/saas/Query.java

```diff
@@ -1034,11 +1034,11 @@ public ExactOnSingleWordQuery getExactOnSingleWordQuery()
 
     private static final String KEY_ALTERNATIVES_AS_EXACT = "alternativesAsExact";
 
-    public @NonNull Query setAlternativesAsExact(List<AlternativesAsExact> types) {
-        if (types == null || types.size() == 0) {
+    public @NonNull Query setAlternativesAsExact(AlternativesAsExact[] types) {
+        if (types == null || types.length == 0) {
```

Comment:
For the sake of symmetry, I think we should allow zero-length arrays (resulting in an empty string value). Because the getter will return a zero-length array for the empty string value.